autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>use double-quotes throughout, as convention. there is use of mixed quotes (single-quote usage on some lines, double-quote usage on other lines)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
done

Reply:
only used single quotes within an f-string wrapped around. a print statement. Was getting errors with double quotes. 
print( f"FinalAutoMLJobObjectiveMetricName: {best_candidate['FinalAutoMLJobObjectiveMetric']['MetricName']}").
Unless there is a different way but double quotes around "MetricName" and "FinalAutoMLJobObjectiveMetric" gives me errors.